### ISSUE TITLE
Provide mini (live or not) previews of terminals when closing the application

### ISSUE THREAD

**Issue by daneel3001** [microsoft/terminal#11422]
It would be useful when closing the application that mini previews are displayed of all of the tabs we will (possibly inadvertently) close. 
Having live previews could be nice to have.

**Comment by zadjii-msft** [comment]
This isn't a terrible idea, though it would likely be a good amount of work. The closeOnExit warning could totally host these previews whenever it's about to close more than one tab (though, it doesn't make sense for multiple panes, they'd be visible in the tab you're closing). Thanks!

**Comment by daneel3001** [comment]
I agree with multi panes it could be less useful but as a consistent behaviour maybe having these previews showing up in all cases on exit could be useful.
Another sub feature could be to be able to in addition cancel out the close the tabs through seeing their preview.
